packages/shared/src/components/modals/CommentBox.tsx

```diff
@@ -91,77 +94,81 @@ function CommentBox({
+            commentBoxClassNames,
           )}
-          ref={commentRef}
-          placeholder="Write your comment..."
-          onInput={(e) => onInput(e.currentTarget.value)}
-          onKeyDown={handleKeydown}
-          onClick={onInputClick}
-          onPaste={onPaste}
-          tabIndex={0}
-          aria-label="New comment box"
-          aria-multiline
+        >
+          <header className="flex items-center mb-2">
+            <RoundedImage
+              imgSrc={authorImage}
+              imgAlt={`${authorName}'s profile`}
+              background="var(--theme-background-secondary)"
+            />
+            <div className="flex flex-col ml-2">
+              <div className="truncate typo-callout">{authorName}</div>
+              <time
+                dateTime={publishDate.toString()}
+                className="text-theme-label-tertiary typo-callout"
+              >
+                {commentDateFormat(publishDate)}
+              </time>
+            </div>
+          </header>
+          <Markdown content={contentHtml} />
+        </article>
+        <div className="flex items-center px-2 h-11">
+          <div className="ml-3 w-px h-full bg-theme-divider-tertiary" />
+          <div className="ml-6 text-theme-label-secondary typo-caption1">
+            Reply to{' '}
+            <strong className="font-bold text-theme-label-primary">
+              {authorName}
+            </strong>
+          </div>
+        </div>
+        <div className="flex relative flex-1 pl-2">
+          <ProfilePicture user={user} size="small" />
+          <div
+            className={classNames(
+              'ml-3 pr-2 flex-1 text-theme-label-primary bg-transparent whitespace-pre-line border-none caret-theme-label-link break-words typo-subhead resize-none',
+              styles.textarea,
+            )}
+            ref={commentRef}
+            contentEditable
+            role="textbox"
+            aria-multiline
+            aria-placeholder="Write your comment..."
+            onInput={(e) => onTextareaInput(e.currentTarget.innerText)}
+            onKeyDown={handleKeydown}
+            onClick={onInputClick}
+            onPaste={onPaste}
+            tabIndex={0}
+            aria-label="New comment box"
+          />
+        </div>
+        <RecommendedMentionTooltip
+          elementRef={commentRef}
+          offset={offset}
+          mentions={mentions}
+          selected={selected}
+          query={mentionQuery}
+          onMentionClick={onMentionClick}
         />
-      </div>
-      <RecommendedMentionTooltip
-        elementRef={commentRef}
-        offset={offset}
-        mentions={mentions}
-        selected={selected}
-        query={mentionQuery}
-        onMentionClick={onMentionClick}
-      />
-      <div
-        className="my-2 mx-3 text-theme-status-error typo-caption1"
-        style={{ minHeight: '1rem' }}
-      >
-        {errorMessage && <span role="alert">{errorMessage}</span>}
+        <div
```

Comment:
Do we always want to show this empty div?
I see we use it as the margin offset, so it's fine, but might not be best use-case?

Reply:
Was thinking about that as well. Probably it was placed before so the UI won't jumpy when it shows? Sounds like not a big issue for me UX-wise. Should we conditionally render this now? What do you think?